README.md

```diff
@@ -1 +1,116 @@
 # python-esiclient
+
+`python-esiclient` extends the OpenStack CLI to provide simplified user workflows, encapsulating multiple OpenStack CLI commands into a single CLI command.
+
+## `openstack esi node network <command>`
+
+These commands manage network connections to nodes.
+
+### `openstack esi node network list`
+
+List node/network attachments.
+
+```
+   openstack esi node network list
+       [--node <node>]
+       [--network <network>]
+```
+
+**--node <node>**: Filter by node (name or UUID)
```

Comment:
I don't think these are rendering the way you want. They end up all on one line of text. Maybe you want a bullet list?

```
- **--node <node>**: FIlter by node (name or UUID)
- **--network <network>**: Filter by network (name or UUID)
```

Although I would probably opt for "code" rather than "emphassis", as in:

```
- `--node <node>`: Filter by node (name or UUID)
- `--network <network>`: Filter by network (name or UUID)
```

Reply:
Whoops - fixed as well.